api/controllers/articles.controller.js

```diff
@@ -70,3 +73,15 @@ exports.patchArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id: id } = req.params;
+  deleteCommentByID(id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
nice take a look into res.SendStatus() could save you needing an empty .send()

Reply:
Cheers for the tip